src/store/account.js

```diff
@@ -180,26 +180,32 @@ export default {
 			context.getters.timeline.setStore(context).initialFetch();
 		},
 
-		// Fetch data from the SDK By Address.
+		// Fetch data from the SDK By plain Address.
```

Comment:
Should be `Fetch account data from the SDK by address, publicKey or alias namespaceName